package.json

```diff
@@ -31,9 +41,12 @@
   "homepage": "https://github.com/shelljs/shx#readme",
   "devDependencies": {
     "ava": "^0.11.0",
+    "babel-cli": "^6.6.5",
+    "babel-preset-es2015": "^6.6.0",
     "eslint": "^1.10.3",
     "eslint-config-airbnb": "^5.0.1",
-    "eslint-plugin-react": "^3.16.1"
+    "eslint-plugin-react": "^3.16.1",
+    "watch": "^0.17.1"
```

Comment:
`watch` is great, but is a little slow.  They are using the default `watchFile` interval of 5007ms right now :S.  There is a [PR](https://github.com/mikeal/watch/pull/84) and I've pointed it out for fixing.  If they don't fix it soon, we can fork.